doc/rados/troubleshooting/cpu-profiling.rst

```diff
@@ -61,7 +61,7 @@ To reset ``oprofile``, execute the following command::
 	sudo opcontrol --reset   
    
 .. important:: You should reset ``oprofile`` after analyzing data so that 
-   you do not commingle results from different tests.
+   you do not comingle results from different tests.
```

Comment:
One "m" appears to be prior art, but two seems to be the more accepted spelling today, so I suggest that this not be changed.

Reply:
This was an error the commit wasn't supposed to do that 7670b1e94e